app/Resources/views/topedits/result_article.html.twig

```diff
@@ -44,1670 +52,19 @@ G{% extends 'base.html.twig' %}
-                            <td>Matthewrbowker</td>
-                            <td>(<a href="//en.wikipedia.org/w/index.php?title=List_of_space_stations&amp;diff=prev&amp;oldid=490819132" >diff</a>)</td>
-                            <td>(<a href="//en.wikipedia.org/w/index.php?title=List_of_space_stations&amp;action=history&amp;year=2012&amp;month=05 " >hist</a>)</td>
-                            <td style="text-align:right;padding-right:5px;color:green" >159</td>
-                            <td style="font-size:85%" >/* De-orbited space stations */ Added citation</td>
-                        </tr>
-
-                        <tr>
-                            <td style="font-size:95%; white-space:nowrap;">2012-05-05, 17:33  &middot; </td>
-                            <td>Matthewrbowker</td>
-                            <td>(<a href="//en.wikipedia.org/w/index.php?title=List_of_space_stations&amp;diff=prev&amp;oldid=490815728" >diff</a>)</td>
-                            <td>(<a href="//en.wikipedia.org/w/index.php?title=List_of_space_stations&amp;action=history&amp;year=2012&amp;month=05 " >hist</a>)</td>
-                            <td style="text-align:right;padding-right:5px;color:green" >25</td>
-                            <td style="font-size:85%" >added [[Category:Space lists]] using [[WP:HC|HotCat]]</td>
-                        </tr>
-
-                        <tr>
-                            <td style="font-size:95%; white-space:nowrap;">2012-05-05, 17:33  &middot; </td>
-                            <td>Matthewrbowker</td>
-                            <td>(<a href="//en.wikipedia.org/w/index.php?title=List_of_space_stations&amp;diff=prev&amp;oldid=490815654" >diff</a>)</td>
-                            <td>(<a href="//en.wikipedia.org/w/index.php?title=List_of_space_stations&amp;action=history&amp;year=2012&amp;month=05 " >hist</a>)</td>
-                            <td style="text-align:right;padding-right:5px;color:green" >1,446</td>
-                            <td style="font-size:85%" >Added several citations</td>
-                        </tr>
-
-                        <tr>
-                            <td style="font-size:95%; white-space:nowrap;">2012-05-05, 17:03  &middot; </td>
-                            <td>Matthewrbowker</td>
-                            <td>(<a href="//en.wikipedia.org/w/index.php?title=List_of_space_stations&amp;diff=prev&amp;oldid=490811617" >diff</a>)</td>
-                            <td>(<a href="//en.wikipedia.org/w/index.php?title=List_of_space_stations&amp;action=history&amp;year=2012&amp;month=05 " >hist</a>)</td>
-                            <td style="text-align:right;padding-right:5px;color:green" >10</td>
-                            <td style="font-size:85%" >Adding Inuse</td>
+                            <td style="font-size:95%; white-space:nowrap;">{{ rev.timestamp }} &middot; </td>
+                            <td>{{ username }}</td>
```

Comment:
Done. Although I'm not sure why there's a 'username' column here at all because the whole table is for one user (we seem to not be supporting the pipe-separated system of the old tool?)